spec/features/allocate_feature_spec.rb

```diff
@@ -19,55 +20,64 @@
 
   let!(:case_information) {
     create(:case_information, offender: build(:offender, nomis_offender_id: nomis_offender_id), tier: 'A', case_allocation: 'NPS', probation_service: 'England')
-    create(:case_information, offender: build(:offender, nomis_offender_id: never_allocated_offender))
+    create(:case_information, offender: build(:offender, nomis_offender_id: never_allocated_offender_id))
   }
 
   before do
     signin_spo_user
   end
 
-  scenario 'accepting the recommended POM type', vcr: { cassette_name: 'prison_api/create_new_allocation_feature' } do
-    visit prison_prisoner_staff_index_path('LEI', nomis_offender_id)
+  context 'when a journey begins on the "Make new allocations" page' do
+    before do
+      visit unallocated_prison_prisoners_path('LEI')
+
+      # Takes you to the Review case and allocate page where users review case details and scroll down to select from a list of POMs
+      click_link unallocated_offender_name
+      expect(page).to have_current_path(prison_prisoner_staff_index_path('LEI', never_allocated_offender_id))
+    end
 
-    expect(page).to have_content('Determinate')
+    scenario 'accepting the recommended POM type', :js, vcr: { cassette_name: 'prison_api/create_new_allocation_feature' } do
+      expect(page).to have_content('Determinate')
 
-    # Allocate to the Probation POM called "Moic Integration-Tests"
-    within '#recommended_poms' do
-      within row_containing 'Moic Integration-Tests' do
-        click_link 'Allocate'
+      # Allocate to the Prison POM called "Moic Pom"
+      within '#recommended_poms' do
+        within row_containing 'Moic Pom' do
```

Comment:
When we talked it through, we were both confused as to why the recommended POM type had changed from wanting a Probation POM to now wanting a Prison POM...